Algorithms/Searching/Upper_Bound.cpp

```diff
@@ -0,0 +1,55 @@
+//Name : Shivam Singhal
+//Email ID : shivamsinghal0610@gmail.com
+
+/*program to find minimum position of element greater than given value 
+in sorted fashion*/
+
```

Comment:
Abstain from using multiline comments.

Reply:
Why though ? Is there any mention of such a thing in the guidelines ?

Reply:
I think it's mentioned there

Reply:
Yes, made the changes.